web/template/home.gohtml

```diff
@@ -290,8 +292,266 @@
+                                                                                <a href="" download
+                                                                                   class="block text-left hover:bg-gray-100 dark:hover:bg-gray-800 py-2 px-4"
+                                                                                   x-text="file.FriendlyName">
+                                                                                </a>
+                                                                            </template>
+                                                                        </div>
+                                                                    </div>
+                                                                </template>
+                                                            {{end}}
+                                                        </div>
+                                                    </template>
+                                                </div>
+                                            </article>
+                                        </template>
+                                    </article>
+                                    <template x-if="courseStreams.hasNext()">
+                                        <div class="flex flex-grow justify-center mt-8">
+                                            <button class="tum-live-button tum-live-button-secondary" type="button"
+                                                    @click="courseStreams.next()">
+                                                <i class="fa-solid fa-angles-down mr-2"></i>
+                                                More
+                                            </button>
+                                        </div>
+                                    </template>
+                                </section>
+                            </section>
+                        </template>
+                    </div>
+                    <template x-if="plannedStreams.hasElements()">
+                        <section class="tum-live-course-view-item xl:col-span-1">
+                            <header>
+                                <h3>Upcoming</h3>
+                            </header>
+                            <article>
+                                <section class="grid gap-5">
+                                    <template x-for="s in plannedStreams.get()" :key="s.ID">
+                                        <article class="rounded-lg"
+                                                 :class="{'border dark:border-gray-800 dark:bg-gray-800' : s.IsToday()}">
+                                            <template x-if="s.IsToday()">
+                                                <span class="block bg-red-500 text-white w-full px-2 py-px text-xs rounded-t-lg">Today</span>
```

Comment:
I like this today view. Maybe an additional indicator that a waiting room exists would be nice?

Reply:
Currently, the new start page doesn't implement any waiting room logic whatsoever. I thought about writing a reusable component for that. However, I'd rather do that in a separate PR 😅 This one is big enough (I think 🤔).

Reply:
Yes, let's do this in a different PR. I just wanted to mention it, so we don't forget it.